BackofficeBundle/Resources/public/js/backboneRouter.js

```diff
@@ -180,20 +176,6 @@ var OrchestraBORouter = Backbone.Router.extend({
     }); 
   },
 
-  setCurrentMainView: function(view)
```

Comment:
You can also remove the early following code so : 

      // Contains currentMainView, usefull to unbind event when view is changed
      currentMainView: null,